src/seedu/addressbook/AddressBook.java

```diff
@@ -193,7 +193,7 @@
     /**
      * The path to the file used for storing person data.
      */
-    private static String storageFilePath;
+    private static String _storageFilePath;
```

Comment:
Good job following the java coding standard that the module handbook introduced.